helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -669,6 +669,10 @@ otelCollector:
   # existing fields can be disabled by setting them to `null`.
   config: {}
 
+# This Flag allow to add necessary permissions to support all SmartAgent
+# Kubernetes-related plugins and add missing key environmental variables.
+k8sEventsEnabled: false
```

Comment:
This flag should enable the events collection, not just make if possible to configure

Reply:
yes, this flag will only add recommended variables in deployment if it is set as `true`.

Reply:
If this flag is enabled, we need to enable k8s event collection with `smartagent/kubernetes-events` receiver  https://docs.signalfx.com/en/latest/integrations/agent/monitors/kubernetes-events.html. So you need to add that receiver to the configuration and make sure that events are collected from the k8s cluster and can be seen in the Splunk o11y UI

Reply:
Hi @dmitryax 
if this flag is enabled then `smartagent/kubernetes-cluster` receiver will be added to the configuration. and also I have moved the `k8sEventsEnabled` flag under `otelK8sClusterReceiver` as the flag is part of it. and also verified that events are collected from k8s cluster and I can verify in Splunk 011y UI.

Environment Link: https://app.signalfx.com/#/temp/chart/v2/new?toDashboard=E-mi8lpAYAA&mode=graph&sources%5B%5D=host.name:otel-splunk-otel-collector-k8s-cluster-receiver-6f4f54c589kgztm
Access token: `DefaultNew`
Organisation name: `Splunk Playground`
Filters:
- Time frame: past-day
- Metrics name: `kubernetes.deployment.desired` `kubernetes.deployment.available` `kubernetes.namespace_phase`
- Host name: `otel-splunk-otel-collector-k8s-cluster-receiver-6f4f54c589kgztm`
<img width="1777" alt="Screenshot 2021-09-06 at 6 36 28 PM" src="https://user-images.githubusercontent.com/88535006/132225745-09bf2d55-1723-4c2f-bee5-e6393a425a72.png">
<img width="1777" alt="Screenshot 2021-09-06 at 6 36 07 PM" src="https://user-images.githubusercontent.com/88535006/132225767-850b4581-dd8d-4dfe-8775-6ded1cb56200.png">
<img width="1777" alt="Screenshot 2021-09-06 at 6 36 14 PM" src="https://user-images.githubusercontent.com/88535006/132225775-ad87b5a4-4338-48cb-8f8d-8866e2348cbe.png">

Reply:
@nitaliya, the screenshots show k8s cluster metrics that are already being collected. kubernetes-events monitor reports events like pod created etc

Here is how events are shown on the chart:
![Screen Shot 2021-09-07 at 12 10 22 AM](https://user-images.githubusercontent.com/6628631/132300362-ba215fc5-4f4d-4436-919b-fe86915bdba5.png)

Reply:
@dmitryax I have made needful changes in the pipeline and got events exported in Observability UI.
<img width="1792" alt="Screenshot 2021-09-08 at 7 49 08 PM" src="https://user-images.githubusercontent.com/88535006/132526937-db788513-010f-42ed-9167-cccd31bc95ad.png">